server/moving.c

```diff
@@ -26,9 +26,10 @@ void	move_player(
 		    )
 {
   printf("\nmoveplayer, check if player is in map\n");
-  if (player_request->x < 0 || player_request->y < 0
-      || player_request->x > 104 || player_request->y > 88)
-    return;
+  // Commenté sinon on empêche le déplacement.
```

Comment:
Weird, it should only prevent the player to escape the map, so either the logic is wrong either we have a problem x)

Reply:
les datas sont pas bonne et pour le moment ça me bloque plus qu;autre chose